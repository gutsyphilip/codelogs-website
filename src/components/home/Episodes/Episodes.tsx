import React from 'react'
import styles from './episodes.module.scss'
import { SearchInput } from '../../utility/input'

const Episodes: React.FC = () => (
  <section className={styles.episodes}>
      <h6 className={styles.episodes_label}>Episodes</h6>
      <h2 className={styles.episodes_title}>
        Watch videos from past episodes.
      </h2>
      <SearchInput text="Search videos by titles" />

      <div className={styles.episodes_listing}>

      </div>
  </section>
)

export default Episodes
