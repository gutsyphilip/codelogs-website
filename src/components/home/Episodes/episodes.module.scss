@import "../../../styles/breakpoint";
@import "../../../styles/converter";
@import "../../../styles/variables";

.episodes{
    background: $theme-light-purple;
    padding: to-rem(49 40 78 30);
    margin-top: to-rem(82);

    @include md(){
        margin-top: to-rem(85);
    }

    &_label{
        color: $theme-purple;
        font-size: to-rem(14);
    }

    &_title{
        color: $text-dark;
        margin: to-rem(3 0 32 0);
        font-size: to-rem(28);
    }

    &_listing{
        margin-top: to-rem(52)
    }
}