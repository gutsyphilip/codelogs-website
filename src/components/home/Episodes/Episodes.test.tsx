import React from "react"
import renderer from "react-test-renderer"
import Episodes from "./Episodes"

describe("Episodes", () =>{
    it('should render successfully', () =>{
        const tree = renderer.create(<Episodes />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})