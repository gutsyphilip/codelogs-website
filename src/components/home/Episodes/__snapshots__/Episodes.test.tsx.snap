// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Episodes should render successfully 1`] = `
<section
  className="episodes"
>
  <h6
    className="episodes_label"
  >
    Episodes
  </h6>
  <h2
    className="episodes_title"
  >
    Watch videos from past episodes.
  </h2>
  <input
    className="input"
    placeholder="Search videos by titles"
  />
  <div
    className="episodes_listing"
  />
</section>
`;
